features/freaks/build.gradle.kts

```diff
@@ -7,4 +7,7 @@ plugins {
 
 android {
     addProductFlavours(this)
+}
+dependencies {
+    implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1")
```

Comment:
Let's find a way to extract the version number so it will be easier to maintain.